### PR TITLE
Changed read_ebook, borrow_ebook id's on buttons to classes

### LINKED ISSUE #4447: Converting read_ebook and borrow_ebook ids to classes
Currently, all the 'Read' and the 'Borrow' buttons use an `id` of `read_ebook` and `borrow_ebook` respectively. Since it is used almost everywhere on the site, I believe we should convert them to classes instead since `id`s are meant to uniquely identify an element. This will be a big refactor and may involve backend as well.  

### Relevant url?
Any page with the blue Read/Borrow buttons, example: https://openlibrary.org/

### Details
- **Environment (prod/dev/local)?** prod

### Proposal & Constraints

Change the `id` to a `class` instead, resolve all the CSS/ backend errors with the same.

### Stakeholders

I'd talked about this with @cdrini quite a while back.

### PR DESCRIPTION
**Pull request by navneetsaluja**
Merger note: Please merge #4456 after merging this one.
-----

Closes #4447 

refactor

### Technical

Nit changes in the code. 

### Testing

The look of the button and the task it is supposed to do is fine, if there's any weird cases involving the buttons, do let me know.

### Screenshot

On prod: 
Selection_022 
After change: 
Selection_021
[No change in the look] 

Validation errors due to these also go away: 
Before: 
Selection_019
After: 
Selection_020

### Stakeholders

@jdlrobson

### CODE REVIEW

**Comment by navneetsaluja** [comment]
@/jdlrobson
Turned out to be smaller changes than I thought, all it took was to find how the buttons were rendering. Let me know if you have any suggestions/ edits to be made. 
Thanks!

**Comment by jdlrobson** [comment]
Awesome! Bit behind on my reviews but will get to this as soon as i can. Thank you!

File: openlibrary/macros/ReadButton.html [hunk 1]
```diff
@@ -12,14 +12,14 @@
 $ stream_url = "/borrow/ia/%s?ref=ol" % ocaid
 
 <div class="cta-button-group">
-  <a href="$(stream_url)" title="$title" id="$(action)_ebook"
+  <a href="$(stream_url)" title="$title" class="$(action)_ebook cta-btn cta-btn--available"
```
Comment (jdlrobson) [inline openlibrary/macros/ReadButton.html:15]: I'd recommend using the class name  `cta-btn--$(action)` here to keep with the BEM syntax we've been using.

**Review by jdlrobson — CHANGES_REQUESTED** [review]
Minor nitpick with the class name, but this is great!

File: openlibrary/macros/ReadButton.html (hunk 1 above)
Comment (navneetsaluja) [reply openlibrary/macros/ReadButton.html]: Updated, let me know if it looks fine :D

**Review by jdlrobson — APPROVED** [review]
just need to test

File: openlibrary/plugins/openlibrary/js/ol.js [hunk 2]
```diff
@@ -110,7 +110,7 @@ export function initBorrowAndReadLinks() {
     /* eslint-disable no-unused-vars */
     // used in openlibrary/macros/AvailabilityButton.html and openlibrary/macros/LoanStatus.html
     $(document).ready(function(){
-        $('#borrow_ebook,#read_ebook').on('click', function(){
+        $('.cta-btn--borrow,.cta-btn--read').on('click', function(){
```
Comment (jdlrobson) [inline openlibrary/plugins/openlibrary/js/ol.js:113]: @cdrini this is existing code, but am wondering what it's for. It seems like borrow and read are always external links that trigger an HTTP request so this click handler never runs. Any ideas? Can it be removed?

**Pull request merged**